Algorithms/String-Algorithm/Isomorphic.js

```diff
@@ -0,0 +1,21 @@
+s = "abc"
+t = "aaa"
+
+var isIsomorphic = function(s, t) {
+    if (s.length != t.length)
+        return false
+    let m = new Map()
+    for (let i = 0; i < s.length; i++) {
+        if (!m.has(s[i]))
+            m.set(s[i], t[i])
+        else {
+            
+            if (m.get(s[i]) != t[i]) {
+                
+                return false
+            }
+        }
+    }
+    return new Set([...m.values()]).size == m.size
+    
```

Comment:
/*We use a Map to record the key/val pair in S and T. For each char in S, if we never met it before, we record the (s[i], t[i]) pair in map. If we met s[i] before, we compare the value of s[i] with t[i], which are supposed to be the same. After that, we need to check whether map.values() contain duplicate value.*/